common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/rest/RestClientVraPrimitive.java

```diff
@@ -625,7 +624,7 @@ protected VraPackageContent getPackageContentPrimitive(Package pkg) {
     }
     
     private VraPackageContent getPackageContentPrimitive(String packageImportedResponse) {
-        JsonElement root = new JsonParser().parse(packageImportedResponse);
+        JsonElement root = JsonParser.parseString(packageImportedResponse);//new JsonParser().parse(packageImportedResponse);
```

Comment:
Why the comment?

Reply:
commented code is one one... since it is deprecated, i wrote it in valid format. If its ok, i will remove this one too